### ISSUE TITLE
Segfault while parsing a list

### ISSUE THREAD

**Issue by z0mbie42** [phasersec/san-go#1]
```san
ignore = [".git/**"]
run = [ 
  'go run testdata/main.go',
]
```

the second list segfault the lib

**Comment by jpillora** [comment]
Fails on 

```
type S struct {
	A string  `san:"a"`
	B []int64 `san:"b"`
}
```

```
a = "a"
b = [
  1,
  2
]
```

**Comment by z0mbie42** [comment]
Thank you for the report!

I don't have time to investigate today but I will in the next few days!

**Comment by z0mbie42** [comment]
fixed in #df23a4de3e45e1a4bdcaeb100f026a8de0ae61d8 you can update your dependencies :smiley:
